packages/litexa-html/README.md

```diff
@@ -44,25 +46,73 @@ project_dir
         └── html
 ```
 
-## Adding 'mark' SSML Tags
+**WARNING**: Unlike other Litexa extensions which conditionally add the
+interface declaration based on whether or not they were used in your skill, the
+`@litexa/html` extension will always add the HTML interface to your skill. Thus, if you
+do not plan on using them for all skills, it may be best to use the local installation
+option for each project.
+
+## Litexa's Alexa Web API Interface
+
+Integrating your skill with the Alexa Web API will allow it to both send and receive data
+from your web app.
+
+### Skill Manifest
+
+This extension automatically adds the `ALEXA_PRESENTATION_HTML` interface declaration to
+your skill manifest upon deployment.
 
-In your `.litexa` files, you can add `mark` SSML tags to your speech, like so:
+### Usage
+
+#### HTML.isHTMLPresent()
+
+This extension adds a global `HTML` object you can use in both Litexa and code files. It
+has one function, which is to detect whether or not HTML is supported on the user's device
+that the skill is running on. This is best used in conjunction with conditionally sending HTML
+directives, since you may want to fall back to APL if HTML is not supported on a device.
 
 ```coffeescript
 launch
-  say "Hello, World!"
-  HTML.mark("screen:blue")
-  END
+  if HTML.isHTMLPresent()
+    directive htmlStartDirective(webAppUrl) # returns `Alexa.Presentation.HTML.Start` directive; this will launch the web app
+  else
+    apl apl/splashScreen.json # the apl statement conditionally sends the directive based on if the device supports it
+  ...
 ```
 
-This will add a substring like `<mark name="screen:blue"/>` to your response's speech. 
-This is useful for your HTML runtime in that, when it receives the Alexa response,
-the marks can be used to trigger HTML events that interleave with SSML playback.
+**WARNING**: Do not use APL and HTML directives in the same response. The user's device
+will likely experience undesired behavior.
+
+#### Directives
+
+You can use the [`directive`](https://litexa.com/reference/#directive) keyword to add
+HTML directives to your skill responses.
+
+If you're interested in trying out transformers, but want to maintain a skill that can switch
```

Comment:
would hyperlink transformers to official docs, unless that's behind the preview.

Reply:
https://developer.amazon.com/en-US/docs/alexa/alexa-presentation-language/apl-data-source.html#transformers

Reply:
The specifics of transformers in HTML docs are better read through the HTML docs, currently accessible through the request process above.